### DISCUSSION TITLE
Thermostat evolute

### DISCUSSION THREAD

**Discussion by lpuntaloro** [HomeSpan/HomeSpan#676] [Q&A]
Hi, I'm using a homespan thermostat as in the example with a real sensor and it works very well. 
I only had a couple of questions: can you also manage schedules at times? 
Can the intervention threshold (delta ) be adjusted? 
Thanks again for the wonderful work done.

**Comment by HomeSpan** [comment]
Unfortunately HomeKit itself does not have functionality to manage time schedules or create a delta threshold (unlike the Ecobee or Nest applications).  However, in principle you could code this into your sketch logic and access with a separate web interface hosted by HomeSpan.  It would be a good bit of coding but certainly possible.

**Reply by lpuntaloro** [reply]
Yes that's what I do now but using an additional connection with a web server.. is it possible to manage a web page directly inside homespan?

**Reply by HomeSpan** [reply]
Yes, you can definitely add a web page directly inside HomeSpan.  See this [FAQ](https://github.com/HomeSpan/HomeSpan/blob/master/docs/FAQ.md#can-you-add-a-web-server-to-homespan)
